### PR TITLE
added coffeescript example to sample-functions/BaseFunctions

### PR DESCRIPTION
**Pull request by aafrey**
updated Dockerfile and package.json

## Motivation and Context

Adds another language for base function testing

## How Has This Been Tested?

Deployed new base function as a FaaS function and tested functionality through the frontend gateway UI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by alexellis** [comment]
Thanks 👍 Can you edit the README.md too?

**Comment by aafrey** [comment]
README updated

File: sample-functions/BaseFunctions/coffee/build.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+echo "Building functions/base:node-6.9.1-alpine"
+docker build -t functions/base:node-6.9.1-alpine .
```
Comment (alexellis) [inline sample-functions/BaseFunctions/coffee/build.sh]: functions/base:coffee-6.9.1-alpine ?

Comment (aafrey) [inline sample-functions/BaseFunctions/coffee/build.sh]: would you want that labeled with the coffeescript version? i.e. function/base:coffeescript-1.12-alpine?

**Comment by aafrey** [comment]
updated the image name to reflect the coffeescript version. that would probably make the most sense from a coffeescript user perspective. let me know otherwise :)

File: sample-functions/BaseFunctions/coffee/build.sh (hunk 1 above)
Comment (alexellis) [inline sample-functions/BaseFunctions/coffee/build.sh]: Good question. What do you think is best?

**Pull request merged**